ContainerHandling/New-NavContainer.ps1

```diff
@@ -531,6 +536,19 @@ try {
         # When using artifacts, you always use best container os - no need to replatform
         $useBestContainerOS = $false
 
+        if ($artifactUrl -like 'https://bcinsider.blob.core.windows.net/*' -or $artifactUrl -like 'https://bcinsider.azureedge.net/*') {
+            if (!$accept_insiderEULA) {
+                $sasToken = "?$("$($artifactUrl)?".Split('?')[1])"
```

Comment:
Does this work as expected?

Just tested it:
```
>> $artifactUrl = 'https://bcinsider.azureedge.net/sandbox/22.3.11962.0/base'
>> $sasToken = "?$("$($artifactUrl)?".Split('?')[1])"
>> $sasToken
?
```

Consider using the `Uri` object. E.g.:
```
$uri = [Uri] $artifactUrl
$sasToken = $uri.Query # the SAS token is the query of the URI
```

Reply:
Yes, it works as expected - it returns the querypart including a ?
The TestSasToken will give an error if it is a ? (i.e. no sastoken provided)